enterprise/api/v1/views.py

```diff
@@ -308,21 +406,10 @@ def courses(self, request, enterprise_customer, pk=None):  # pylint: disable=inv
 
         Only courses with active course runs are returned. A course run is considered active if it is currently
         open for enrollment, or will open in the future.
-        ---
-        serializer: serializers.CourseSerializerExcludingClosedRuns
         """
         catalog_api = CourseCatalogApiClient(request.user)
         courses = catalog_api.get_paginated_catalog_courses(pk, request.GET)
-
-        # if API returned an empty response, that means pagination has ended.
-        # An empty response can also means that there was a problem fetching data from catalog API.
```

Comment:
@UmanShahzad I think it would make sense to preserve this comment. Please replace "means" with "mean" in L318.